.github/workflows/test.yml

```diff
@@ -71,6 +76,8 @@ jobs:
         go-version: ${{ matrix.go-version }}
 
     - name: build
+      env:
+        BUILDTAGS: seccomp ${{ matrix.dmz }}
```

Comment:
Maybe we should do something like this in Makefile:
```make
BUILDTAGS += EXTRA_BUILDTAGS
```
and use `EXTRA_BUILDTAGS` here, so we don't have to repeat the default build tags here.